modules/user/user.graphql.js

```diff
@@ -36,6 +37,7 @@ input UserInput {
     comments: [ID]
     banned: Boolean
     confirmed: Boolean
+    token: String
```

Comment:
Why do we need it here? Why can user input token by himself?

Reply:
removed